src/helper/site-utils.php

```diff
@@ -517,7 +518,11 @@ function set_postfix_files( $site_url, $site_service_dir ) {
 function configure_postfix( $site_url, $site_fs_path ) {
 
 	chdir( $site_fs_path );
-	EE::exec( \EE_DOCKER::docker_compose_with_custom() . ' exec postfix postconf -e \'relayhost =\'' );
+
+	EE::exec( \EE_DOCKER::docker_compose_with_custom() . " exec php sh -c 'echo \"host postfix\ntls off\nfrom no-reply@$site_url\" > /etc/msmtprc'" );
```

Comment:
Add option to configure `no-reply@$site_url` as well from env var.
Check if `REPLY_EMAIL` is set in `.env`, if set, use it otherwise, fallback to the default `no-reply@$site_url`.

Reply:
@mrrobot47 Done.